dev/bots/prepare_package.dart

```diff
@@ -601,14 +601,12 @@ class ArchivePublisher {
     if (dest.endsWith('.json')) {
       mimeType = 'application/json';
     }
-    final List<String> args = <String>[];
-    // Use our preferred MIME type for the files we care about
-    // and let gsutil figure it out for anything else.
-    if (mimeType != null) {
-      args.addAll(<String>['-h', 'Content-Type:$mimeType']);
-    }
-    args.addAll(<String>['cp', src, dest]);
-    return await _runGsUtil(args);
+    return await _runGsUtil(<String>[
+      // Use our preferred MIME type for the files we care about
+      // and let gsutil figure it out for anything else.
+      if (mimeType != null) ...<String>['-h', 'Content-Type:$mimeType'],
+      ...<String>['cp', src, dest],
```

Comment:
nit: maybe this could just be a regular list of strings?  
```
'cp',
src,
dest,
```

Reply:
Done.

I kept the same semantic as the original code. Technically I guess compiler could optimized it to be the same as what you propose.